src/Atm_led.cpp

```diff
@@ -1,6 +1,6 @@
 #include "Atm_led.hpp"
 
-Atm_led& Atm_led::begin( int attached_pin, bool activeLow ) {
+Atm_led& Atm_led::begin( int attached_pin, bool activeLow, byte* blinkPattern, int blinkPatternSize ) {

```

Comment:
Won't this *require* that one supply the third and fourth argument? What about adding another method with this, so people who don't want to use the blink pattern can do so like before?